admin-dev/themes/new-theme/js/app/utils/datepicker.js

```diff
@@ -27,12 +27,35 @@ import 'url-polyfill';
 const $ = global.$;
 
 /**
- * Enable all datepickers.
+ * Initializes all datetimepickers.
+ *
+ * 'data-format' attribute is optional to define custom date-time format.
+ * If provided format contains time, the time picker appears aside date picker calendar.
+ * Default datepicker icons are overridden in css by appending context to corresponding class.
+ *
+ * Example usage in template:
+ *
+ * <div class="input-group datepicker">      // .datepicker class is used to select datetimepicker object
+ *   <input type="text" class="form-control"
+ *     data-format="YYYY-MM-DD HH:mm:ss"     // provide data-format attr in case you need custom format
+ *   />
+ * </div>
  */
 const init = function initDatePickers() {
```

Comment:
We need a more accurate name for this class, because we might need multiple date-pickers extensions. Some with YYYY-MM-DD, some with only YYYY-MM, some side-by-side, some other displays ... so we need to name them accurately in order to avoid confusion and pick the one we need

Reply:
Actually I would like to wait for #14748 this to be solved, as I guess it will modify this js file.

Reply:
#14748 has been fixed by https://github.com/PrestaShop/PrestaShop/pull/14914 😉